src/Sarif/Autogenerated/SarifLog.cs

```diff
@@ -31,10 +31,7 @@ public SarifNodeKind SarifNodeKind
             }
         }
 
-        /// <summary>
-        /// The URI of the JSON schema corresponding to <see cref="Version"/>.
-        /// </summary>
-        [DataMember(Name="$schema")]
+        [DataMember(Name = "$schema", IsRequired = false, EmitDefaultValue = false)]
```

Comment:
The mapping is now generated automatically. The doc comment needs to be added to the schema.